### ISSUE TITLE
Jump start scrolling

### ISSUE THREAD

**Issue by hypothesist** [philc/vimium#2224]
When I long press `j`, the page automatically starts scrolling smoothly. What I'd like instead is something to emulate the action of long pressing the down arrow - the page doesn't immediately start scrolling down. I've got used to that so much that the long press `j` feels a bit weird.

I know not everyone will want this, so maybe a off-by-default option?

Thanks for the plugin anyway :) Pretty amazing stuff :+1:

**Comment by sco-tt** [comment]
Have you tried turning of the option 'Use Smooth Scrolling' under 'Miscellaneous Options'. I'm not sure if it's exactly what you're looking for, but it does feel closer to the down arrow in my (admittedly brief) test.

**Comment by hypothesist** [comment]
Not really. What I meant was something emulating chrome's behaviour when you long press the down arrow - it smoothly scrolls down a few pixels, then waits for around half a second before restarting the continuous smooth scroll. That delay feels a little easier on my eye for some reason, and I'd love to have it as an option.

**Comment by smblott-github** [comment]
I guess the scrolling effect you describe is just an artefact of how keyboard repeat works.  It's not obvious that the UX is better or worse in either case, so I'd say stick with the status quo.  And this probably isn't above threshold for a toggle on the options page.

**Comment by gdh1995** [comment]
On my Windows, the keyboard will generate `repeat: true` keydown events about every 33ms, only after a stop of about 500ms from the first `repeat: false` keydown event (without any keydown event in this period).

In my options, smooth scroll should not suspend in such a period, otherwise it becomes weird and my eyes will get tired soon, because of the strange changing scroll speed.

**Comment by philc** [comment]
Closing; I don't we'll do this, and I like the current UX.
